### ISSUE TITLE
Adding features

### ISSUE THREAD

**Issue by payallenka** [SyedImtiyaz-1/Code-Shop#11]
I would like to enhance the homepage i.e, the home page design by adding the bestsellers feature (it will display the bestselling shirts for example).
A customer review page.
Kindly assign this issue to me.

**Comment by Khushi-Gupta13** [comment]
I will try to add review section to the project where happy customers can write about their experience while exploring the website so kindly assign the project to me as well with @payallenka .

**Comment by payallenka** [comment]
yes @SyedImtiyaz-1 it will be great if you assign it to us both. I wish to collaborate with @Khushi-Gupta13

**Comment by SyedImtiyaz-1** [comment]
Done!!
